src/status_im2/contexts/activity_center/notification/common/view.cljs

```diff
@@ -16,3 +20,111 @@
       :text-style     style/user-avatar-tag-text}
      (activity-center.utils/contact-name contact)
      (multiaccounts/displayed-photo contact)]))
+
+(defn- render-swipe-action
+  [{:keys [active-swipeable
+           extra-fn
+           interpolation-opacity
+           interpolation-translate-x
+           on-press
+           swipe-button
+           swipeable-ref]}]
+  (fn [^js _progress ^js drag-x]
```

Comment:
why _progress with `_` ?

Reply:
I knew you wouldn't like it :) It's just "theoretically" there to help the developer understand what this function can receive, but yeah, let's say farewell to `_progress`.